src/main/groovy/com/athaydes/spockframework/report/template/TemplateReportCreator.groovy

```diff
@@ -155,9 +155,10 @@ class TemplateReportCreator implements IReportCreator {
     protected void handleUnrolledFeature( FeatureRun run, FeatureInfo feature, Closure callback ) {
         run.failuresByIteration.eachWithIndex { iteration, problems, index ->
             final name = Utils.featureNameFrom( feature, iteration, index )
-            final result = problems.any( Utils.&isError ) ? 'ERROR' :
-                    problems.any( Utils.&isFailure ) ? 'FAILURE' :
-                            Utils.isSkipped( feature ) ? 'IGNORED' : 'PASS'
+            final errors = problems.any( Utils.&isError )
+            final failures = problems.any( Utils.&isFailure )
+            final isSkipped = Utils.isSkipped( feature )
```

Comment:
This is a little bit prettier, but makes all checks be performed every time, instead of only the needed checks as it was before. Can you verify that on a large test suite, this has a very insignificant impact on performance?